ignite/services/chain/serve.go

```diff
@@ -148,31 +150,29 @@ func (c *Chain) Serve(ctx context.Context, cacheStorage cache.Storage, options .
 					if c.served {
 						c.served = false
 
-						fmt.Fprintln(c.stdLog().out, "💿 Saving genesis state...")
+						c.ev.Send("💿 Saving genesis state...")
 
 						// If serve has been stopped, save the genesis state
 						if err := c.saveChainState(context.TODO(), commands); err != nil {
-							fmt.Fprint(c.stdLog().err, err.Error())
+							c.ev.Send(err.Error())
 							return err
 						}
 
 						genesisPath, err := c.exportedGenesisPath()
 						if err != nil {
-							fmt.Fprintln(c.stdLog().err, err.Error())
 							return err
 						}
-						fmt.Fprintf(c.stdLog().out, "💿 Genesis state saved in %s\n", genesisPath)
+						c.ev.Send(fmt.Sprintf("💿 Genesis state saved in %s", genesisPath))
 					}
 				case errors.As(err, &buildErr):
-					fmt.Fprintf(c.stdLog().err, "%s\n", errorColor(err.Error()))
+					c.ev.Send(colors.Error(err.Error()))
```

Comment:
If we add view package `pkg/view/builderrorview` I see it will import `"github.com/ignite-hq/cli/ignite/chainconfig"` because it is needed to check that error is actually a validation error. 
```go
var validationErr *chainconfig.ValidationError
if errors.As(err, &validationErr) {
  // do something
}
```

Let's leave this possible enhancement for the future PRs.

Reply:
Solved in 2079781